src/System.Resources.ResourceWriter/src/System/Resources/ResourceWriter.cs

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// 
+
```

Comment:
nit - since you're going to resubmit this anyway, can you cleanup these blank lines at the top of the cs files